### DISCUSSION TITLE
post/bsp-dungeon

### DISCUSSION THREAD

**Discussion by giscus** [foopod/blog6#14] [General]
# post/bsp-dungeon

https://jonoshields.com/post/bsp-dungeon

**Comment by King-Friendly** [comment]
Thanks so much! This put the problem so simply, and it was incredibly easy to understand! I appreciate you greatly :)

**Comment by LucyThePirate** [comment]
The way you write is very informative, I like that you walk through how everything works rather than just posting your finished code. Thank you!
